explorer/client/src/components/displaybox/DisplayBox.tsx

```diff
@@ -19,6 +23,23 @@ function DisplayBox({ display }: { display: string }) {
 
     useEffect(() => {
         setHasFailedToLoad(false);
+        setHasImgBeenChecked(false);
+        setImgAllowState(false);
+
+        new ImageModClient()
+            .checkImage(transformURL(display))
+            .then(({ ok }) => {
+                //setHasImgBeenChecked(true);
+                setImgAllowState(ok);
+            })
+            .catch((error) => {
+                //setHasImgBeenChecked(true);
+                // default to allow, so a broken img check service doesn't break NFT display
+                setImgAllowState(true);
```

Comment:
The current code means that when the image check service breaks, people will be able to post and see dodgy images. It would be better if there were instead a message saying that the content moderation service was down, a message that replaces any images.

The service is currently down due to a CORS error that can be observed in the terminal. This means that, as it stands, the content moderation is not having an effect. Users can post and observe dodgy content.
![image](https://user-images.githubusercontent.com/11377188/175514957-87fd9343-2408-442b-bf00-4173218e52ea.png)

Reply:
The same approach applies here. We can use a default image.